src/constants/constants.js

```diff
@@ -589,15 +589,19 @@ export const BLOCK_TIME_MILLIS = {
 
 export const TRANSACTION_CONFIRMATIONS = 1;
 
+export const CACHE_DATA_TYPES = {
```

Comment:
Not sure I get why would we need this

Reply:
Still don't think it's needed, for the specific events we have predefined types in the ABI

e.g. for `ParametersStorage` it can't be anything else except `Number`, so I would just cast values based on the types in the ABI
<img width="342" alt="image" src="https://github.com/OriginTrail/ot-node/assets/71610423/910e03e4-2be5-4364-86f6-ed78a2ed898e">